indico/modules/events/abstracts/forms.py

```diff
@@ -74,6 +74,29 @@ def validate_contrib_type_required(self, field):
             raise ValidationError(_('The event has no contribution types defined.'))
 
 
+class AbstractReviewingSettingsForm(IndicoForm):
+    """Settings form for abstract reviewing"""
+
+    num_answers = IntegerField(_('Number of choices'), [NumberRange(min=2), DataRequired()],
+                               description=_('The number of choices reviewers have for each question.'))
+    scale_lower = IntegerField(_('Scale (from)'), [InputRequired()])
+    scale_upper = IntegerField(_('Scale (to)'), [InputRequired()])
+    conveners_final_judgment = BooleanField(_('Allow track conveners to judge'), widget=SwitchWidget(),
```

Comment:
`conveners_judment`. There's only one type of judgment. I would even go for `allow_conveners_to_judge`.